spec/models/asset_spec.rb

```diff
@@ -211,26 +211,48 @@
   end
 
   describe '#accessible_by?' do
-    it 'returns true if the asset is not draft' do
-      asset = FactoryBot.build(:asset, draft: false)
-      expect(asset).to be_accessible_by(nil)
-    end
+    context 'asset is live' do
+      let(:asset) { FactoryBot.build(:asset, draft: false) }
 
-    it 'returns true if the asset is draft but not access limited' do
-      asset = FactoryBot.build(:asset, draft: true, access_limited: [])
-      expect(asset).to be_accessible_by(nil)
+      it 'returns true' do
+        expect(asset).to be_accessible_by(nil)
+      end
     end
 
-    it 'returns true if the asset is draft and access limited and the user is authorised to view it' do
-      user = FactoryBot.build(:user, uid: 'user-id')
-      asset = FactoryBot.build(:asset, draft: true, access_limited: ['user-id'])
-      expect(asset).to be_accessible_by(user)
+    context 'asset is a draft thats access_limited' do
+      let(:asset) do
+        FactoryBot.build(
+          :asset, draft: true, access_limited_user_ids: ['user-id'], access_limited_organisation_ids: ['org-id']
+)
+      end
+
+      it 'returns true if users id is authorised to view it' do
```

Comment:
Minor, but we usually go to double quotes in order to be grammatical ;-).

Reply:
fixed!